R/mergeSEs.R

```diff
@@ -565,9 +565,9 @@ setMethod("right_join", signature = c(x = "ANY"),
     
     # Create a LinkDataFrame based on the link data
     links <- LinkDataFrame(
-        nodeLab = links[["nodeLab"]],
+        node.lab = links[["node.lab"]],
         nodeNum = links[["nodeNum"]],
-        nodeLab_alias = links[["nodeLab_alias"]],
+        node.lab_alias = links[["node.lab_alias"]],
```

Comment:
Not sure if this is working. links variable is a LinksDataFrame from TreeSE, and it has these columns that are already named